[2024-11-10T06:27:35.935+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-10T06:27:35.947+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stock_Data_ETL.load_stock_data manual__2024-11-10T06:27:30.986480+00:00 [queued]>
[2024-11-10T06:27:35.950+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stock_Data_ETL.load_stock_data manual__2024-11-10T06:27:30.986480+00:00 [queued]>
[2024-11-10T06:27:35.950+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-10T06:27:35.955+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_stock_data> on 2024-11-10 06:27:30.986480+00:00
[2024-11-10T06:27:35.960+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=615) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-10T06:27:35.959+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Stock_Data_ETL', 'load_stock_data', 'manual__2024-11-10T06:27:30.986480+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/FinanceDataAnalytics_ETL_Pair9.ipynb.py', '--cfg-path', '/tmp/tmp1uf3fqhj']
[2024-11-10T06:27:35.961+0000] {standard_task_runner.py:91} INFO - Job 132: Subtask load_stock_data
[2024-11-10T06:27:35.961+0000] {standard_task_runner.py:63} INFO - Started process 618 to run task
[2024-11-10T06:27:35.980+0000] {task_command.py:426} INFO - Running <TaskInstance: Stock_Data_ETL.load_stock_data manual__2024-11-10T06:27:30.986480+00:00 [running]> on host 2538ecb7b4d3
[2024-11-10T06:27:36.022+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='shatayu.thakur@sjsu.edu' AIRFLOW_CTX_DAG_OWNER='Pair 9' AIRFLOW_CTX_DAG_ID='Stock_Data_ETL' AIRFLOW_CTX_TASK_ID='load_stock_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T06:27:30.986480+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T06:27:30.986480+00:00'
[2024-11-10T06:27:36.023+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-10T06:27:36.027+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-11-10T06:27:36.027+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.11-linuxkit-aarch64-with-glibc2.36
[2024-11-10T06:27:36.028+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-10T06:27:36.643+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-10T06:27:37.105+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-10T06:27:37.106+0000] {logging_mixin.py:188} INFO - Target Table Initialized and Ready to Store Data from ETL
[2024-11-10T06:27:37.106+0000] {logging_mixin.py:188} INFO - Loading Apple stock data...
[2024-11-10T06:28:15.082+0000] {logging_mixin.py:188} INFO - Loading Nvidia stock data...
[2024-11-10T06:28:53.467+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-10T06:28:53.469+0000] {logging_mixin.py:188} INFO - Data load complete for Apple and Nvidia stocks.
[2024-11-10T06:28:53.470+0000] {connection.py:762} INFO - closed
[2024-11-10T06:28:53.510+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2024-11-10T06:28:53.569+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-10T06:28:53.571+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-10T06:28:53.586+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Stock_Data_ETL, task_id=load_stock_data, run_id=manual__2024-11-10T06:27:30.986480+00:00, execution_date=20241110T062730, start_date=20241110T062735, end_date=20241110T062853
[2024-11-10T06:28:53.863+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-10T06:28:53.884+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-10T06:28:53.885+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
